libs/deal-ticket/src/components/market-info/tooltip-mapping.tsx

```diff
@@ -0,0 +1,103 @@
+import { t } from '@vegaprotocol/react-helpers';
+import { Link } from '@vegaprotocol/ui-toolkit';
+import type { ReactNode } from 'react';
+
+export const tooltipMapping: Record<string, ReactNode> = {
+  makerFee: t(
+    'Maker portion of the fee is transferred to the non-aggressive, or passive party in the trade (the maker, as opposed to the taker).'
+  ),
+  liquidityFee: t(
+    'Liquidity portion of the fee is paid to market makers for providing liquidity, and is transferred to the market-maker fee pool for the market.'
```

Comment:
```suggestion
    'Liquidity portion of the fee is paid to market makers for providing liquidity, and is transferred to the liquidity fee pool for the market.'
```

Reply:
@MadalinaRaicu This is not coming from generated docs, correct?

Reply:
correct. it has been manually added in another ticket.